### DISCUSSION TITLE
Korean characters are broken in SVG Chart.

### DISCUSSION THREAD

**Discussion by hangum** [eclipse-birt/birt#1276] [Q&A]
version : birt 4.5.0

All Hangul is fine on my Mac. 

However, on Docker or AWS Linux, Hangul is broken. 
It is broken in Word, PDF, and Excel.  
![image](https://user-images.githubusercontent.com/1335915/236752017-59043fc8-655e-43d0-b3db-3abb9e621c90.png)

English comes out fine.

Where to look?
Thank you.

**Comment by hvbtup** [comment]
I cannot really help you, but I can give you some pointers.

 * First of all, please try BIRT 4.13. I don't know if it will help, but it is worth a try. 

 * Note that BIRT uses some XML files `fontsConfig*.xml` which define where to look for TrueType font files.

 * Note that the *SVG* charts are rendered by Apache Batik. I'm not sure, but perhaps this does *not* use the `fontsConfig*.xml` files and instead uses Javas builtin logic to search for fonts.

**Comment by hangum** [comment]
@hvbtup 
Thanks for the answer.

Engine Setting
1.  Font directory settings
FontFactory.registerDirectory(_httpRequest.getServletContext().getRealPath("/fonts"));

2. Image resolutions
config.getAppContext().put(EngineConstants.APPCONTEXT_CHART_RESOLUTION, 600);

Copy the fonts you use to your server.
org.eclipse.birt.runtime_4.5.0/fontsConfig.xml
ex) /usr/share/fonts/
